extensions/flyway/deployment/src/main/java/io/quarkus/flyway/FlywayDevConsoleProcessor.java

```diff
@@ -73,7 +73,7 @@ protected void handlePostAsync(RoutingContext event, MultiMap form) throws Excep
                 Path migrationDir = path.resolve(config.locations.get(0));
                 Files.createDirectories(migrationDir);
                 Path file = migrationDir.resolve(
-                        "V1.0.0__" + curateOutcomeBuildItem.getEffectiveModel().getAppArtifact().getArtifactId() + ".sql");
+                        "V1.0.0__" + curateOutcomeBuildItem.getApplicationModel().getAppArtifact().getArtifactId() + ".sql");
```

Comment:
Why this change? /cc @aloubyansky

Reply:
This is the recommended method in the deprecated javadoc